test/e2e/operator_test.go

```diff
@@ -56,6 +60,31 @@ func getClient() (client.Client, string, error) {
 	return kubeClient, namespace, nil
 }
 
+func newClusterIngress(name, ns, domain string) *ingressv1alpha1.ClusterIngress {
+	return &ingressv1alpha1.ClusterIngress{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "ingress.openshift.io/v1alpha1",
+			Kind:       "ClusterIngress",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+		},
+		Spec: ingressv1alpha1.ClusterIngressSpec{
+			IngressDomain: &domain,
+			HighAvailability: &ingressv1alpha1.ClusterIngressHighAvailability{
+				Type: ingressv1alpha1.CloudClusterIngressHA,
```

Comment:
Might want a TODO here that the test needs to be infrastructure/platform aware in the very near future (maybe when we move it over to the origin extended tests).